src/test/java/edu/harvard/hul/ois/fits/junit/FitsOutputTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2009 Harvard University Library
+ *
+ * This file is part of FITS (File Information Tool Set).
+ *
+ * FITS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FITS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with FITS.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package edu.harvard.hul.ois.fits.junit;
+
+import edu.harvard.hul.ois.fits.FitsMetadataElement;
+import edu.harvard.hul.ois.fits.FitsOutput;
+import edu.harvard.hul.ois.fits.tests.AbstractLoggingTest;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FitsOutputTest extends AbstractLoggingTest {
+
+    @Test
+    public void testFitsOutput() throws Exception {
+
+        FitsOutput fitsOutput = new FitsOutput(fitsMetadata);
+        FitsMetadataElement element = fitsOutput.getMetadataElement("fileinfo");
```

Comment:
The output from line 33 is immediately overwritten by the output of line 34. Is this line needed?

Reply:
This line should have been removed. It's now gone.